cluster-up/cluster/kind-1.27-sriov/provider.sh

```diff
@@ -44,12 +51,13 @@ function configure_registry_proxy() {
     KIND_BIN="$kind_binary_path" PROXY_HOSTNAME="$ci_proxy_hostname" $configure_registry_proxy_script
 }
 
-function up() {
-    # print hardware info for easier debugging based on logs
-    echo 'Available NICs'
-    ${CRI_BIN} run --rm --cap-add=SYS_RAWIO quay.io/phoracek/lspci@sha256:0f3cacf7098202ef284308c64e3fc0ba441871a846022bb87d65ff130c79adb1 sh -c "lspci | egrep -i 'network|ethernet'"
-    echo ""
+function deploy_sriov() {
```

Comment:
I would call it something like `prepare_sriov_cluster` because it does more than deploy SR-IOV components.

Reply:
imho deploy verb encapsulate more than preparation, which stands usually for the first phase of something